blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -591,7 +608,7 @@ def create_bvh_tree_multi_objects(mesh_objects: List["MeshObject"]) -> mathutils
     return bvh_tree
 
 
-def compute_poi(objects: List["MeshObject"]) -> np.ndarray:
+def compute_poi(objects: List[MeshObject]) -> np.ndarray:
```

Comment:
Are you sure this works? As far as I know you cannot reference the class inside the class in the type hints?

Reply:
Also I think the `typing.Self` attribute will only be available starting with python 3.11

Reply:
True, blender is only on 3.10 right now. :( Self is much nicer. 

PS: Line 611 is not inside the class it is after the class definition and that should work